Hello.

longjunyu2 opened the pull request:
使用AndroidStudio进行构建，能够编译通过，没有修改核心代码。

longjunyu2 committed:
The Android Studio rebuilding project was built by Gradle. Complete compilation is achieved through Android Studio and NDK.实现在AndroidStudio下就可以完整编译

longjunyu2 committed:
Create README.md

longjunyu2 committed:
Update README.md

New README updated.

longjunyu2 committed:
修正库:改为使用zhuowei修改的组件，包括aFileChooser,OpenAL-Soft

longjunyu2 committed:
Update README.md

longjunyu2 committed:
Update README.md

longjunyu2 committed:
新资源:创建目录放置一些经过修改的，可以直接通过Boardwalk启动的Minecraft游戏文件。

longjunyu2 committed:
新资源:创建图片预览。修改README.md

longjunyu2 committed:
Update README.md

longjunyu2 committed:
修正:FileUtils.getFile(uri)参数为一个而不是两个。

FileUtils.getFile(uri)，由于使用了新的aFileChooser，部分代码有变化。

zhuowei commented:
@longjunyu2 Great work! I haven't tested this yet - does this work on Android 8 and above?

Also, this is a really large pull request - may I ask why you decided to merge the openal-soft and aFileChooser repos into this one?

Edit:

- The Minecraft jars shouldn't be included.
- I saw your comment about LWJGL in the readme. 
I'm not sure what's the cleanest way to include LWJGL. I've never managed to integrate its build; I've always built LWJGL by hand and copied it into the repo.
I don't feel it's a good idea to include an LWJGL binary .jar in the repo, though. Would be interested to hear your ideas on this.
- The predexed libraries probably shouldn't be included: I think it adds too many files to this repo.
in the old build system I just had shell scripts that [compiles](https://github.com/zhuowei/Boardwalk/blob/master/predex_build.sh) and [runs](https://github.com/zhuowei/Boardwalk/blob/master/predex_run.sh) the predexer. The predexer then downloads the libraries Minecraft needs, and runs `dex` on them.
I ran that by hand: if you want, you could try integrating that into gradle somehow. (Maybe have gradle call the scripts? I know very little about the new build system.)

longjunyu2 committed:
修复:Lwjgl库 native libs库缺失

longjunyu2 committed:
新增 加入SD卡写入动态提权

解决 Android 6.0 以及以上系统 Boardwalk不能获取存储写入权限而导致启动崩溃的问题。
解决 Android API 27 下黑屏问题，现在Boardwalk可以正常使用了。

longjunyu2 committed:
新增 加入SD卡写入动态提权

解决 Android 6.0 以及以上系统 Boardwalk不能获取存储写入权限而导致启动崩溃的问题。
解决 Android API 27 下黑屏问题，现在Boardwalk可以正常使用了。
引入v7支持
